portal/src/components/v2/ImageInput/ImageInput.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useCallback, useRef } from "react";
+import styles from "./ImageInput.module.css";
+
+import placeholderIcon from "../../../images/image_input_placeholder_icon.svg";
```

Comment:
I am wondering if we are using Radix icons. Or this is an exception of using a custom svg as icon?

Reply:
This icon is not in radix icon.

Reply:
But I've checked the icons list again and found that if we remove the background, we can use a radix icon for it. Let me change this part.

Reply:
I overlooked that several svg icons were introduced in the Step 2 PR. For example, the password_icon seems to be the `input` icon of Radix icons.

Reply:
For the list of login methods, there are always items not in the radix icons list. e.g. All oauth items. You don't find them in the pr because they are not implemented yet.
I can ask if designer can change the icon for email, phone and username to a radix icon.

Reply:
1. Use Radix icons whenever possible
2. Introduce a new component to render backgrounded icon
3. Use either fontawesome tabler-icons for brand icons

Reply:
Pending reply:
https://www.figma.com/design/K38RiF42gekApCdtwRLF4W?node-id=7260-45333#1231854133

Reply:
Follow up in https://linear.app/authgear/issue/DEV-2664/use-radix-icon-or-existing-fonts-for-icons